src/WebhookClientServiceProvider.php

```diff
@@ -23,16 +23,20 @@ public function packageBooted()
     {
         Route::macro('webhooks', function (string $url, string $name = 'default', $method = 'post') {
 
-            if(! in_array($method, ['get', 'post', 'put', 'patch', 'delete'])) {
+            if (! in_array($method, ['get', 'post', 'put', 'patch', 'delete'])) {
                 throw InvalidMethod::make($method);
             }
 
+            if (config('webhook-client.add_unique_token_to_route_name', false)) {
+                $name .= Str::random(8);
```

Comment:
@it-can I think we need to add the dot prefix here in order for this to work.

Reply:
aah missed that one... will fix